dashboards/aerospike/aerospike.json

```diff
@@ -0,0 +1,1037 @@
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Cluster \n### Cluster Integrity:\nFalse i.e. 0, some nodes are faulty; true i.e. 1, cluster is complete.\n\n### Cluster Member: \nFalse i.e. 0, node is not joined; true i.e. 1, node is joined.\n\n"
+          }
+        },
+        {
+          "title": "Cluster Integrity",
+          "layout": {
+            "column": 4,
+            "row": 2,
+            "width": 3,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT latest(aerospike_node_stats_cluster_integrity) as ''"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 0
+              },
+              {
+                "alertSeverity": "WARNING",
+                "value": 1
+              }
+            ]
+          }
+        },
+        {
+          "title": "Cluster Member",
+          "layout": {
+            "column": 7,
+            "row": 2,
+            "width": 3,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT latest(aerospike_node_stats_cluster_is_member) as ''  "
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 0
+              },
+              {
+                "alertSeverity": "WARNING",
+                "value": 1
+              }
+            ]
+          }
+        },
+        {
+          "title": "Time Since the Last Reclustering Event (seconds)",
+          "layout": {
+            "column": 10,
+            "row": 2,
+            "width": 3,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT max(aerospike_node_stats_time_since_rebalance) as ''"
```

Comment:
Try to convert the result as a Hours, Minutes and Seconds.

Reply:
As per the manohar suggestion, results convert into Minutes